src/ApiConnectors/ApiOptions.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace PhpTwinfield\ApiConnectors;
+
+/**
+ * Class ApiOptions
+ * @package PhpTwinfield\ApiConnectors
+ */
+final class ApiOptions
+{
+    /**
+     * @var string[] exception messages that should be retried
```

Comment:
The naming of the variable is perfect so the comment is superfluous, I'd remove it.